src/niivue/index.ts

```diff
@@ -1805,6 +1805,25 @@ export class Niivue {
     return MESH_EXTENSIONS.includes(ext)
   }
 
+  /**
+   * Remove volume by url
+   * @param buffer - ArrayBuffer with the entire contents of a mesh and volume
+   * @param name - string of filename, extension used to infer type (NIfTI, MGH, MZ3, etc)
+   * @see {@link http://192.168.0.150:8080/features/draganddrop.html | live demo usage}
+   */
+
+  async addFromArrayBuffer(buffer: ArrayBuffer, name: string): Promise<void> {
```

Comment:
@hanayik I tried that approach first. However, the HTML file open dialog does not directly return a file object and for privacy reasons the file name does not provide the file path (as it would reveal details about the users directory structure). Therefore, I think I have provided the minimal example to load a file from the HTML `input` element.

To allow you to confirm that there are no easier methods, I have updated my commit so that when you run the hot-reloadable `npm run dev` you will have a web page with a file open dialog.